extensions/wikia/Insights/Insights.setup.php

```diff
@@ -62,6 +62,22 @@
 $wgAutoloadClasses['InsightsModuleController'] = $IP.'/skins/oasis/modules/InsightsModuleController.class.php';
 
 /**
+ * Api Querys
+ */
+$wgAutoloadClasses[ 'ApiQueryUnconvertedInfoboxes' ] = $dir . 'ApiQueryUnconvertedInfoboxes.class.php';
```

Comment:
Should be moved to extensions/wikia/Insights/InsightsUnconvertedInfoboxes.setup.php